similaritymeasures/similaritymeasures.py

```diff
@@ -879,3 +879,111 @@ def dtw_path(d):
     path = np.array(path)
     # reverse the order of path, such that it starts with [0, 0]
     return path[::-1]
+
+def mae(exp_data, num_data, p=2, threshold=1000000):
+    """
+    Compute the Mean Absolute distance.
+    This computes the mean of absolute values of the distances 
+    between 2 corresponding points on the curve. This can use all 
+    distance metrics that are available in scipy.spatial.distance.cdist.
+
+    Parameters
+    ----------
+    exp_data : array_like
+        Curve from your experimental data. exp_data is of (M, N) shape, where
+        M is the number of data points, and N is the number of dimmensions
+    num_data : array_like
+        Curve from your numerical data. num_data is of (P, N) shape, where P
+        is the number of data points, and N is the number of dimmensions
+    p: float,
+        1 <= p <= infinity. Which Minkowski p-norm to use.
+    threshold : positive int
+        If M * N * K > threshold, algorithm uses a Python loop instead of large temporary arrays.
+
+    Returns
+    -------
+    r : float
+        MAE.
+    """
+    c = distance_matrix(exp_data, num_data, p=p, threshold=threshold)
```

Comment:
Sorry, `distance_matrix` doesn't do what I thought. It's another `O(n*n)` operation, when we want a `O(n)` operation.

Change this line to
```python
c = np.abs(exp_data - num_data)
```